packages/ezstarrating_extension/ezextension/ezstarrating/classes/ezsrratingobjecttreenode.php

```diff
@@ -35,11 +35,13 @@
 class ezsrRatingObjectTreeNode extends eZContentObjectTreeNode
 {
      /**
-     * Construct
+     * Construct, shouldn't be called directly
+     * Use {@link ezsrRatingObjectTreeNode::makeObjectsArray()} to create new objects.
      * 
+     * @access protected
      * @param array $row
      */
-    protected function __construct( $row )
+    public function __construct( $row )
     {
         $this->eZContentObjectTreeNode( $row );
```

Comment:
This one still works?

Reply:
You are correct, it won't work. I'll create another PR to fix it.

Reply:
Another, why not in the same here. topic: legacy 2017.10 and php7 compatibility